lib/anyplayer/players/xmms2.rb

```diff
@@ -68,22 +68,20 @@ def platforms
     [:unix, :linux]
   end
 
-
   private
 
   def xmms2(command)
     ret = %x(xmms2 #{command}).split("\n").first
-    ret ? ret.strip : ""
+    ret ? ret.strip : ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.